common/app/views/fragments/nav/editionPicker.scala.html

```diff
@@ -0,0 +1,19 @@
+@()(implicit request: RequestHeader)
+
+@import common.{Edition, LinkTo}
+
+<div class="new-header__edition-picker">
+    @fragments.inlineSvg("world", "icon")
+
+    <select onchange="location = this.value;">
```

Comment:
Can you add a hidden `<label>` to this to help assistive technology users work out what this does?